### ISSUE TITLE
tiscript databing  doesn't work with go-sciter.

### ISSUE THREAD

**Issue by TongxiJi** [sciter-sdk/go-sciter#210]
https://sciter.com/model-view-whatever-the-plus-engine-for-sciter/
this's  not working for in go-sciter.
It works fine  with sciter.exe and sciter-mfc.exe.

code below:
`
	w, err := window.New(sciter.SW_MAIN,sciter.DefaultRect)
	if err != nil {
		log.Fatal(err)
	}
   	w.LoadFile(`x:\\0-basic-variable-binding.htm`)
	w.SetTitle("Hello, world")
	w.Show()
	w.Run()
`

**Comment by VanillaScent** [comment]
I think this is because of as stated in the README.md 

> Things that are not supported:
> 
> Sciter Node API
> TIScript Engine API

**Comment by pravic** [comment]
> Things that are not supported

No, it is about the native API bindings. It is parallel to TIScript itself which should work as is anywhere.

**Comment by big-hippo** [comment]
i use tis +plus in many projects with go-sciter, it works well.
please use inspector app and debug it
